### PR TITLE
Addind Generic 404 page error

### PR DESCRIPTION
**Pull request by essymo**
adding the line;
+ config.exceptions_app = self.routes in (config/application.rb)  to tell the application's router to handle the errors. If the URL is missing or misspelled then it should redirect to the 404 page.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false
```
Comment (equivalentideas) [inline config/environments/development.rb]: Why have you changed this as part of this PR @essymo ?

File: app/views/home/_about.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ A huge thank you to [Google Australia](http://www.google.com.au) whose donation
 
 ### <a name='contribute'></a>You can help
 
-Make this project better for everyone by [summarising divisions](/divisions) and [maintaining and discussing policies](/policies). You can also [make a tax deductible donation](https://www.openaustraliafoundation.org.au/donate/theyvoteforyou/) to the OpenAustralia&nbsp;Foundation to support us in creating and maintaining projects like this.
+Make this project better for everyone by [summarising divisions](/help/research/) and [maintaining and discussing policies](/policies). You can also [make a tax deductible donation](https://www.openaustraliafoundation.org.au/donate/theyvoteforyou/) to the OpenAustralia&nbsp;Foundation to support us in creating and maintaining projects like this.
```
Comment (equivalentideas) [inline app/views/home/_about.md]: This is a good change but it's not related to the other changes in this PR, you should remove this commit from the branch. I'm happy to explain how to do this if you like :)

File: config/application.rb [hunk 3]
```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.exceptions_app = self.routes
```
Comment (equivalentideas) [inline config/application.rb]: Nice one @essymo It's looks like this bit was missed when the pages were added in d2799fe25c7352c90b75611e69dcae8431118b9b

File: config/environments/development.rb (hunk 1 above)
Comment (essymo) [inline config/environments/development.rb]: Hi @equivalentideas  My bad..The purpose of that line was to test the behavior locally it should be changed I guess.

File: app/views/home/_about.md (hunk 2 above)
Comment (essymo) [inline app/views/home/_about.md]: Please do,always ready to learn new stuff :) . How do I go about it?

**Comment by essymo** [comment]
Hi @equivalentideas I sent a cleaner pull request  for the two commits using different branches kindly review and maybe could close this particular pull request. :)

**Pull request closed without merging**